src/types/gitlab.d.ts

```diff
@@ -5,6 +5,8 @@ declare namespace Gitlab {
 
   type MergeRequestState = 'all' | 'closed' | 'locked' | 'merged' | 'opened';
 
+  type MergeRequestAction = 'open' | 'close' | 'reopen' | 'update' | 'approved' | 'unapproved' | 'approval' | 'unapproval' | 'merge;'
```

Comment:
```suggestion
  type MergeRequestAction = 'open' | 'close' | 'reopen' | 'update' | 'approved' | 'unapproved' | 'approval' | 'unapproval' | 'merge';
```

Reply:
Wow surprised I didn't notice this error 😬 Good eyes!